build-android.gradle

```diff
@@ -0,0 +1,54 @@
+buildscript {
+  repositories {
+    jcenter()
+    google()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.1.0'
+  }
+}
+
+apply plugin: 'com.android.library'
+
+repositories {
+  jcenter()
+  google()
+}
+
+group = 'com.hierynomus'
+
+dependencies {
+  implementation "org.slf4j:slf4j-api:1.7.25"
```

Comment:
Let's extract the version numbers to `gradle.properties` so that they're not duplicated (and become impossible to maintain)

Reply:
Fixed